docs/sdk/api/syncActions.md

```diff
@@ -402,11 +402,11 @@ client.execute(taxCategoriesRequests)
 
 > From package [@commercetools/sync-actions](/sdk/api/README.md#sync-actions).
 
-Creates a [sync action](/sdk/Glossary.md#sync-action) that allows to build API update actions for _zones_.
+Creates a [sync action](/sdk/Glossary.md#syncaction) that allows to build API update actions for _zones_.
 
 #### Arguments
 
-1.  `actionGroups` _(Array)_: A list of [action group](/sdk/Glossary.md#sync-action) in case some actions need to be _blacklisted_ or _whitelisted_.
+1.  `actionGroups` _(Array)_: A list of [action group](/sdk/Glossary.md#syncaction) in case some actions need to be _blacklisted_ or _whitelisted_.
```

Comment:
I don't see how these changes are related.. 😞

Reply:
Just realised they had been wrong all along so included a little fix for it.